pygmt/src/fitcircle.py

```diff
@@ -0,0 +1,127 @@
+"""
+fitcircle - Fit coordinators to create vectors on a sphere.
```

Comment:
I find the description from the official documentation is easier to understand.
```suggestion
fitcircle - Find mean position and great [or small] circle fit to points on sphere.
```